apps/total-typescript/src/templates/tip-template.tsx

```diff
@@ -212,63 +231,81 @@ const TipOverlay: React.FC<{tips: Tip[]}> = ({tips}) => {
       <div className="z-20 lg:absolute left-0 top-0 w-full h-full flex flex-col items-center justify-center text-center leading-relaxed text-lg p-5">
         {/* <ShareTip lesson={tip} /> */}
         <div className="grid lg:grid-cols-3 sm:grid-cols-2 grid-cols-1 gap-2 h-full ">
-          {take(shuffle(tips), 9).map((tip) => {
-            const video = tip?.resources.find(
-              (resource: SanityDocument) => resource._type === 'videoResource',
-            )
-
-            const thumbnail = `https://image.mux.com/${video.muxAsset.muxPlaybackId}/thumbnail.png?width=288&height=162&fit_mode=preserve`
-
-            return (
-              <button
-                onClick={() => {
-                  router
-                    .push({
-                      pathname: '/tips/[tip]',
-                      query: {tip: tip.slug.current},
-                    })
-                    .then(() => {
-                      handlePlay()
-                    })
-                }}
-                className="aspect-video w-full z-0 group font-medium text-left text-gray-200 relative leading-tight h-full flex items-end justify-start bg-gray-900/60 rounded-lg p-8"
-              >
-                <div className="flex flex-col relative z-10">
-                  <span className="text-xs font-semibold text-gray-500 pb-1 font-mono uppercase">
-                    Tip
-                  </span>
-                  <span className="font-medium">{tip.title}</span>
-                </div>
-                <Image
-                  src={thumbnail}
-                  alt=""
-                  aria-hidden="true"
-                  layout="fill"
-                  className="object-cover opacity-30 blur-xs z-0 group-hover:opacity-40 group-hover:brightness-150 transition"
-                  quality={100}
-                />
-                <div
-                  className="absolute w-full h-full left-0 top-0 flex items-start justify-end p-5"
-                  aria-hidden="true"
-                >
-                  <PlayIcon className="w-10 h-10 flex-shrink-0 text-gray-300 group-hover:scale-100 scale-50 group-hover:opacity-100 opacity-0 transition" />
-                </div>
-              </button>
-            )
-          })}
+          {take(shuffle(tips), 9).map((tip) => (
+            <VideoOverlayTipCard suggestedTip={tip} />
+          ))}
         </div>
       </div>
     </div>
   )
 }
 
+const VideoOverlayTipCard: React.FC<{suggestedTip: Tip}> = ({suggestedTip}) => {
```

Comment:
extracted this out so it's a FC instead of a "callback" so it can use hooks directly